src/python/foglamp/data_purge/logs.json

```diff
@@ -0,0 +1 @@
+{"2017-07-11 13:45:18": {"rowsRemoved": 0, "complete": "2017-07-11 13:45:18", "failedRemovals": 0, "unsentRowsRemoved": 0, "startTime": "2017-07-11 13:45:18", "rowsRemaining": 0}}
```

Comment:
@Terris-Dianomic - You are right. This shouldn't be checked in the official release. The reason for the checkin of it (now) is so that whomever reviews the branch will see what output will look like.